doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html

```diff
@@ -84,6 +84,8 @@ <h2 id="Overview">Overview</h2>
 				</td>
 				<td>
 					<p><ul>
+<li>When using custom scripts, some arguments may be modified in live. This can cause issues when using paths starting with a "/" .If it occurs, try removing the leading "/" from the paths used as arguments.</li>
```

Comment:
:) 
`starting with a "/" .If it occurs`
Shall be:
`starting with a "/". If it occurs`

Reply:
Oops,
done correctly this time I hope :p